dev-docs/source/index.rst

```diff
@@ -25,6 +25,16 @@ Guides
 :doc:`GettingStarted`
    Describes the process of obtaining and building the mantid code base
 
+.. toctree::
```

Comment:
Can we leave then out of the toctree for now? There needs to be a final task to create the main page once all of the pages have been translated anyway. Don't worry about the warnings it will create for now.